widgetsnbextension/src/widget_output.js

```diff
@@ -5,19 +5,18 @@
 // This widget is strongly coupled to the notebook because of the outputarea
 // dependency.
 var widgets = require("@jupyter-widgets/controls");
+var outputWidgets = require("@jupyter-widgets/output");
 var _ = require("underscore");
 require('./widget_output.css');
 
 var outputArea = new Promise(function(resolve, reject) {
         requirejs(["notebook/js/outputarea"], resolve, reject)
 });
 
-var OutputModel = widgets.DOMWidgetModel.extend({
-    defaults: _.extend({}, widgets.DOMWidgetModel.prototype.defaults, {
-        _model_name: "OutputModel",
-        _view_name: "OutputView",
+var OutputModel = outputWidgets.OutputModel.extend({
+    defaults: _.extend({}, outputWidgets.OutputModel.prototype.defaults(), {
         msg_id: "",
-        outputs: [],
+        outputs: []
```

Comment:
`outputs` should be in the base class, since that is a necessary part of the model (it is assumed by the python side).